src/gui/options/OptionsModel.cpp

```diff
@@ -226,6 +226,17 @@ void OptionsModel::SetPerfectCircle(bool perfectCircle)
 	notifySettingsChanged();
 }
 
+bool OptionsModel::GetMomentumScroll()
+{
+	return Client::Ref().GetPrefBool("MomentumScroll", false);
```

Comment:
Should probably default to `true`, as it does in ScrollPanel.cpp.

Reply:
Yep, got kinda lost in the whole option chain!

Reply:
Yeah, it's a bit awkward. Not for long though, if everything goes according to plan.